src/date/fromString.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Returns a Date from string value format espected format dd[/]mm[/]yyyy
+ * @param {string} dateFrom
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
+import { nullValue } from "../generator/out/not_implemented";
+import { EMPTY_DATE_VALUE } from "./core";
+
+export const fromString = (dateFrom: string): Date => {
+  const dateParts = dateFrom.match(
```

Comment:
Implementation assumes date format is dd/mm/yyyy. Is it always the case? Couldn't it be using another date format?

Reply:
We need to solve the string representation format for this date values in a model preferences dependant way. 
Issue #33 was created.